app/scripts/contentscript.js

```diff
@@ -37,12 +37,12 @@ if (shouldInjectProvider()) {
  *
  * @param {string} content - Code to be executed in the current document
  */
-function injectScript(content) {
+function injectScript() {
```

Comment:
Should `injectScript(inpageBundle);` become `injectScript();` on line 31?

Reply:
And remove `inpageSuffix` and `inpageBundle` consts?

Reply:
I am thinking of adding a feature flag for MV3 and using it here to conditionally inject script text or source.